### PR TITLE
Upgrade to Rails 5

### PR DESCRIPTION
**Pull request by tijmenb**
https://trello.com/c/UvadVLm8

### CODE REVIEW

File: config/cable.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+development:
```
Comment (whoojemaflip) [inline config/cable.yml]: can we lose this file plz

File: config/initializers/wrap_parameters.rb [hunk 2]
```diff
@@ -5,10 +5,10 @@
...
 # Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
 ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json] if respond_to?(:wrap_parameters)
+  wrap_parameters format: [:json]
 end
 
 # To enable root element in JSON for ActiveRecord objects.
 # ActiveSupport.on_load(:active_record) do
-#  self.include_root_in_json = true
+#   self.include_root_in_json = true
```
Comment (whoojemaflip) [inline config/initializers/wrap_parameters.rb]: 👾

File: features/step_definitions/policy_steps.rb [hunk 3]
```diff
@@ -197,15 +197,15 @@
...
-          "links" => {
-        "organisations" => [],
-        "lead_organisations" => [],
-        "people" => [],
-        "working_groups" => [working_group_1["content_id"]],
-        "related" => [],
-        "email_alert_signup" => [@policy.email_alert_signup_content_id],
-        "policy_areas" => [],
-      }
+    "links" => {
```
Comment (whoojemaflip) [inline features/step_definitions/policy_steps.rb]: Should this be in a different PR?

Comment (tijmenb) [inline features/step_definitions/policy_steps.rb]: Removed the commit (not worth a separate PR though)

File: config/cable.yml (hunk 1 above)
Comment (tijmenb) [inline config/cable.yml]: Done
